pyrtls.pyi

```diff
@@ -0,0 +1,407 @@
+"""
```

Comment:
What's the use case for repeating the README here? Is that a common convention?

Reply:
This gives a description of the whole module when you hover over its name in an editor.

Reply:
Could we move this into the module itself as a docstring?

Reply:
No, it unfortunately doesn't work.